### ISSUE TITLE
[WCAG 2.2] [Windows Terminal - Multiple Tab]: No single pointer alternative is provided to move the tabs.

### ISSUE THREAD

**Issue by SaurabhNew** [microsoft/terminal#17900]
### Windows Terminal version

1.22.2362.0

### Windows build number

27695.1000

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 27695.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro steps:**
1. Open the 'windows terminal' application.
2. Open Multiple tabs and navigate among the tabs.
3. Now, verify whether any single pointer alternative is provided to move the tabs or not.
5. Observe the issue.

**User experience:**
Users who struggle with performing dragging movements will feel difficulty to move tabs, if there is no single pointer alternative is provided to perform the action.

**WCAG Reference Link:**
[Understanding Success Criterion 2.5.7: Dragging Movements | WAI | W3C](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements)

**Attachment:**
[No single pointer alternative is provided to move the tabs.zip](https://github.com/user-attachments/files/16958526/No.single.pointer.alternative.is.provided.to.move.the.tabs.zip)

### Expected Behavior

A single pointer alternative mechanism should be provided to the users to move the tabs.

### Actual Behavior

User can be able to move the tabs here & there by using mouse, but no alternative method is available for the user to perform action using single pointer.

**Comment by carlos-zamora** [comment]
This is a /dup of #16717.

Please refer to the discussion in that thread for further details.

**Comment by SaurabhNew** [comment]
@carlos-zamora,
Having keyboard shortcut to move the tabs cannot be consider as a single pointer alternative.

Suggestion: A button can be defined in a context menu of tabs using which users can move the tab.

**Comment by zadjii-msft** [comment]
Is there precedent for something like this in Edge?

Expanding out "Move tab to new window" to "Move Tab > {To new window, To the left, To the right, ...}" as a nested menu might be an option here
